tempesta_fw/t/functional/wrk/post_small.lua

```diff
@@ -0,0 +1,3 @@
+wrk.method  = "POST"
+wrk.headers = {["Content-Type"]="text/plain"}
+wrk.body    = "some not very long text\n"
```

Comment:
RFC 7230 3.3.2: `... Content-Length header field is normally sent in a POST request ...`
Ether `Transfer-Encoding` or `Content-Length` should be set for this request or Request will be treated as request without body. Look for the TempestaFW log, you should receive multiple parsing errors for this test.
See RFC 7230 3.3.3 for more details.